Sources/UnleashProxyClientSwift/Client/UnleashProxyClientSwift.swift

```diff
@@ -155,9 +149,12 @@ public class UnleashClientBase {
 @available(iOS 13, tvOS 13, *)
 public class UnleashClient: UnleashClientBase, ObservableObject {
     @MainActor
-    public func start(_ printToConsole: Bool = false) async throws {
+    public func start(
+        bootstrap: Bootstrap = .toggles([]),
```

Comment:
with bootstrap going as a first parameter isn't it gonna be a breaking API change?

Reply:
I don't think it should since it has a default argument. However, if `printToConsole` is used, it now needs to be explicit since without, at the call-site it would be like this:
```swift
// Without explicit printToConsole argument, confusing what the "true" applies to at a glance
await start(bootstrap: .toggles([...]), true)`

// With explicit printToConsole argument, clearer what the bool input applies to
await start(bootstrap: .toggles([...]), printToConsole: true)

// Default arguments so following should work
await start() // default: no bootstrap toggles, and printToConsole false

await start(bootstrap: .toggles([...])) // boostrapped, default: printToConsole false

await start(printToConsole: true) // default: no bootstrap toggles, override: printToConsole
```

Reply:
fair point